### DISCUSSION TITLE
Limit issue

### DISCUSSION THREAD

**Discussion by SakshiTharkude** [apache/superset#20189] [Ideas]
Please increase the limit values , which is manually done in the chart section when I select the bar graph chart
but default values in the select tab of limit would make it more flexible and people may use it for large datasets including many columns 
Limit should be extended , like as of now it is 1000000.
It should exceed more than 10L

**Comment by rusackas** [comment]
If we increase it too much, we'll likely run into performance issues of various types. It does, however, seem plausible to make this into something you could configure in `config.py` - would that be good enough? Would you have any interest in opening a PR? Moving this to a discussion to see if others have interest here as well.

**Reply by SakshiTharkude** [reply]
Yeah sure i also want to know the procedure to configure config.py.
So that we could make changes according to our requirements.
